src/fromJEOL.js

```diff
@@ -37,7 +38,7 @@ export function fromJEOL(buffer) {
   //newInfo.comment = info.comment;
   newInfo.solvent = info.solvent;
   newInfo.temperature = info.temperature.magnitude;
-  newInfo.probeName = info.probeId;
+  newInfo.probeName = info.probeName || 'undefined';
```

Comment:
Why defining a undefined string ? I would rather leave it empty or put an empty string.